07LessonSeven/in-class-project/css/my-style.css

```diff
@@ -0,0 +1,48 @@
+* {
+    font-family: Tahoma, Geneva, sans-serif;
+}
+.nasa-background {
+    background-image: url("../img/nasa-background.jpg");
+    height: 450px;
+    width: 850px;
+    color: white;
+    text-align: center;
+    font-size: 50px;
+}
+h1 {
+    padding: 194px;
```

Comment:
try giving your h1 a margin of 0 because the default margins are interfering with the placement of the other sections.